src/app/scripts/cac/map/cac-map-control.js

```diff
@@ -350,6 +361,9 @@ CAC.Map.Control = (function ($, Handlebars, cartodb, L, turf, _) {
             zoomControl.addTo(map);
             initializeOverlays();
             initializeLayerControl.apply(this, null);
+            if (currentLocationMarker) {
+                currentLocationMarker.addTo(map);
+            }
```

Comment:
Nothing in the console.  I'm forcing the location using Chrome developer tools, and my debugger inside the `map.on('locationfound', function(event) {` function does trip, but no dot.